<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>taskControlClient</class>
 <widget class="QMainWindow" name="taskControlClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>taskControlClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Connection</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Serwer IP address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_RTserverIPaddress1">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>158</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_RTserverIPaddress2">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>75</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_RTserverIPaddress3">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_RTserverIPaddress4">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>55</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_serverSelection">
            <item>
             <property name="text">
              <string>cosmos</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>szajcomp</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Serwer port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_RTserverPort">
            <property name="maximum">
             <number>64000</number>
            </property>
            <property name="value">
             <number>3333</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Processes</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_checkStatus">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>170</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>170</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>119</red>
            <green>119</green>
            <blue>119</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>119</red>
            <green>119</green>
            <blue>119</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>119</red>
            <green>119</green>
            <blue>119</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>generate trajectory</string>
      </property>
      <property name="text">
       <string>Check status</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+C</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>540</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_serverSelection</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>taskControlClient</receiver>
   <slot>server_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>server_changed()</slot>
 </slots>
</ui>
